scripts/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+cd /home/agent
+wget https://github.com/estuary/flow/releases/download/dev/flow-x86-linux.tar.gz
+mkdir -p /home/agent/.bin
+tar -xvf flow-x86-linux.tar.gz -C /home/agent/.bin 
```

Comment:
sanity check: the Flow image places binaries in `/usr/local/bin`

This is placing them in `/home/agent/.bin`. I'm guessing due to permissions that `/usr/local/bin` isn't writable 👍 

[Meanwhile ops is currently using](https://github.com/estuary/ops/blob/master/lib/agent/agent.libsonnet#L33) `/usr/local/bin` as well. You're planning to change it there after this is landed?

Reply:
All of your assumptions are correct.